src/sparseml/pytorch/torchvision/train.py

```diff
@@ -533,6 +535,10 @@ def collate_fn(batch):
     # load params
     if checkpoint is not None:
         if "optimizer" in checkpoint and not args.test_only:
```

Comment:
I think we can safely just add `args.resume` to this and that should handle transfer learning case?
```suggestion
        if args.resume and "optimizer" in checkpoint and not args.test_only:
```